.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+<!-- 
+Prior to submitting a PR, review the CONTRIBUTING.md document for recommendations on how to test:
+https://github.com/wasmerio/wasmer/blob/master/CONTRIBUTING.md#pull-requests
+-->
+
+# Description
+<!-- 
+Provide details regarding the change including motivation,
+links to related issues, and the context of the PR.
+-->
+
```

Comment:
We should mention updating the changelog!

Reply:
I added a comment about updating the changelog.